pkg/operator/ceph/cluster/mgr/spec_test.go

```diff
@@ -105,7 +106,8 @@ func TestHostNetwork(t *testing.T) {
 		DataPathMap:  config.NewStatelessDaemonDataPathMap(config.MgrType, "a", "rook-ceph", "/var/lib/rook/"),
 	}
 
-	d := c.makeDeployment(&mgrTestConfig)
+	d, err := c.makeDeployment(&mgrTestConfig)
+	assert.Nil(t, err)
```

Comment:
NoError() is preferred instead of Nil() when checking an error. Please review other asserts for the same pattern.
```suggestion
	assert.NoError(t, err)
```

Reply:
Done.